src/amo/components/CategoryIcon/styles.scss

```diff
@@ -0,0 +1,151 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+$default-icon-size: 64px;
+
+@mixin icon($name) {
+  display: inline-block;
+  height: $default-icon-size;
+  mask: url("./#{$name}.svg") no-repeat 50% 50%;
+  transform: scale(0.8);
+  width: $default-icon-size;
+
+  // Edge does not support CSS mask, but all other browsers do. This avoids
```

Comment:
I would rephrase this comment: "This prevents Edge from rendering a square block of color because it can't mask over the SVG."